docs/api/structures/web-source.md

```diff
@@ -2,4 +2,3 @@
 
 * `code` string
 * `url` string (optional)
-* `startLine` Integer (optional) - Default is 1.
```

Comment:
Should this get a release note at least? It does change a published API, though in a backwards-compatible (I think?) way.